assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -131,4 +131,21 @@ export const getDomElementOffset = (el) => {
   const scrollTop = getDocumentScrollTop();
   const scrollLeft = window.pageXOffset || document.documentElement.scrollLeft || document.body.scrollLeft || 0;
   return { top: rect.top + scrollTop, left: rect.left + scrollLeft };
+};
+
+export const setDomElementOffset = (el, offset) => {
```

Comment:
I need this function to compute the new absolute top. (dom offsets are weird)
But I could modify it to return an object with the new top and use react state + style prop instead of 'elem.style.top = smth' you're right

Reply:
Or not, since I need the dom element to compute the top I need to set.. Will look into it.